README.md

```diff
@@ -59,6 +59,70 @@ f1.content = "The quick brown fox jumped over the lazy dog."
 bf1.save
 ```
 
+## Extending Directly Contained Derivatives
+
+Hydra Works comes with the ability to attach [Hydra Derivatives](https://github.com/projecthydra/hydra-derivatives) to a GenericFile. This allows a GenericFile of an image, video, audio file, or document type to directly contain appropriate derivative files such as thumbnail, extracted_text, mp3, ogg, mp4, and webm. The derivatives are configured in `lib/hydra/works/models/concerns/generic_file/contained_files.rb` and `lib/hydra/works/models/concerns/generic_file/derivatives.rb`.
+
+More directly contained derivatives can be configured for the Persist Derivatives service by extending `contained_files` and `makes_derivatives`, as long as the processor is defined in Hydra Derivatives.
+
+### contained_files
+```
+# lib/hydra/works/models/concerns/generic_file/contained_files.rb
```

Comment:
I would greatly prefer the derivative definitions be in the app's models directory or in an initializer, instead of monkey-patching.  I'm not opposed to monkey-patching when the module is out of your control or can't/won't be changed for some reason.  But that's not the case here and we should be able to provide a cleaner extension mechanism.